assets/css/style.css

```diff
@@ -357,6 +397,7 @@ footer {
 }
 #metadata {
   border: 1px solid #cacaca;
+  
```

Comment:
Need to watch out for unneeded whitespace. Will be implementing coding style guide soon.